### DISCUSSION TITLE
Packaging and installation NAppGUI

### DISCUSSION THREAD

**Discussion by LPeco** [frang75/nappgui_src#144] [General]
Hello, I have a question related to packaging and installation, since when I run the command:

cmake ­­install build ­­config Release ­­prefix C:/nappgui

The structure of folders and files does not match the one described in the manual, for example, libraries with the extension .a do not appear, instead, .lib appears.

Am I doing something wrong?

Thank you very much.

**Comment by frang75** [comment]
Hi @LPeco. Thanks for using nappgui.
In Windows, when you compile with VisualStudio, the static library name format is `XXXX.lib`, instead of `libXXXX.a` used in Unix-Like systems and MinGW. You also will see a `cmake` folder. I'll update the documentation.

**Reply by LPeco** [reply]
Ok, Thank you so much.
